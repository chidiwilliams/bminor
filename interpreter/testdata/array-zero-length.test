b: array [0] integer = {};
