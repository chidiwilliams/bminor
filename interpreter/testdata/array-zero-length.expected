err: Error at '0' on line 1: expect length of array type expression to be a positive integer
